src/client/components/HomepageHeader/Header.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import robot from '../../assets/images/robot.png';
+import './header.css';
+import PropTypes from 'prop-types';
+
+function Header({ color, size }) {
```

Comment:
Could consider naming `size` to `fontSize` instead as it is the fontSize you change with this an not the size of the header :)